### ISSUE TITLE
TQDM.ipynb reference raises exception: AttributeError: Column.value property descriptor does not exist

### ISSUE THREAD

**Issue by MarcSkovMadsen** [holoviz/panel#2554]
I'm on Panel 0.12.0, Param 1.11.1, Bokeh: 2.3.3

If I run the TQDM.ipynb reference notebook top to bottom and click one of the buttons I get a `AttributeError: Column.value property descriptor does not exist` error.

![image](https://user-images.githubusercontent.com/42288570/126439660-554b5d01-57e6-481b-8069-0e3f689c9599.png)

**Comment by MarcSkovMadsen** [comment]
You can replicate the problem using this simple script

```python
import time

import panel as pn

from panel.widgets import Tqdm

pn.extension()

tqdm = Tqdm()

def run_loop(*events, timeout=0.2):
    for i in tqdm(range(0,10), desc="My loop bar", leave=True, colour='#666666'):
        time.sleep(timeout)

run_loop(timeout=0.01)

button = pn.widgets.Button(name="Run Loop", button_type="success")
button.on_click(run_loop)
pn.Row(button, tqdm).servable()
```

**Comment by MarcSkovMadsen** [comment]
I've tried to identify the cause and understand how to fix this without success.

I can see it tries to set the `value` on a `Column` which does not make sense. But why I don't understand.

I will need your help @philippjfr to fix this.

**Comment by philippjfr** [comment]
Got a fix, will push out a 0.12.1 in the coming days.
